tests/chainer_tests/functions_tests/math_tests/test_exponential_m1.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(), (3, 2)],
+}))
+class Expm1FunctionTest(unittest.TestCase):
+
+    def setUp(self):
+        self.x = numpy.random.uniform(.5, 1, self.shape).astype(numpy.float32)
+        self.gy = numpy.random.uniform(-1, 1, self.shape).astype(numpy.float32)
+
+    def check_forward(self, op, op_np, x_data):
+        x = chainer.Variable(x_data)
+        y = op(x)
+        gradient_check.assert_allclose(
+            op_np(self.x), y.data, atol=1e-7, rtol=1e-7)
+
+    def check_forward_cpu(self, op, op_np):
+        self.check_forward(op, op_np, self.x)
```

Comment:
I think it is simpler to merge `check_forward_cpu` and `test_expm1_forward_cpu` into one function.